api-controllers/ServiceBrokerApiController.js

```diff
@@ -265,11 +265,13 @@ class ServiceBrokerApiController extends FabrikBaseController {
     }
 
     function gone() {
-      res.status(CONST.HTTP_STATUS_CODE.GONE).send({});
+      res.status(CONST.HTTP_STATUS_CODE.GONE).send({
+        state: CONST.OPERATION.SUCCEEDED
```

Comment:
According to [service broker api](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#response-1), `GONE` should be sent with empty body. Any particular reason why we are sending body.

Reply:
You are right. I missed this spec. Will change accordingly.